vendor.mod

```diff
@@ -78,37 +79,20 @@ require (
 	go.uber.org/zap v1.10.0 // indirect
 	golang.org/x/sys v0.0.0-20210823070655-63515b42dcdf // indirect
 	golang.org/x/text v0.3.4 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	google.golang.org/genproto v0.0.0-20201110150050-8816d57aaa9a // indirect
-	gopkg.in/fsnotify.v1 v1.4.7 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
-	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.15
-	github.com/akutz/gosync => github.com/akutz/gosync v0.1.0
 	github.com/containerd/containerd => github.com/containerd/containerd v1.5.2
-	github.com/coreos/go-semver => github.com/coreos/go-semver v0.2.0
-	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
 	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
-	github.com/google/uuid => github.com/google/uuid v1.0.0
 	github.com/hashicorp/go-immutable-radix => github.com/tonistiigi/go-immutable-radix v0.0.0-20170803185627-826af9ccf0fe
-	github.com/hpcloud/tail => github.com/hpcloud/tail v1.0.0
-	github.com/matttproud/golang_protobuf_extensions => github.com/matttproud/golang_protobuf_extensions v1.0.1
-	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega => github.com/onsi/gomega v1.5.0
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc95
```

Comment:
Same for this one; if we update containerd, this may not be needed (I see containerd 1.5.x now uses run v1.0.2); https://github.com/containerd/containerd/blob/v1.5.9/go.mod#L43